src/visit_vtk/lightweight/vtkVisItPointLocator.C

```diff
@@ -642,14 +627,14 @@ void vtkVisItPointLocator::FindDistributedPoints(int N, const double x[3],
             minCurrentCount = GetMin(currentCount);         
             if (currentCount[oct] == N)
               {
-              qsort(res[oct], currentCount[oct], sizeof(idsort),private_vtkidsortcompare);
+              std::sort(res[oct], res[oct]+currentCount[oct]);
```

Comment:
did use of qsort cause a warning?

Reply:
There was a warning that would have required casts. I guess I got a little tired of they way the casts made the qsort call  hard to read (there are many of them in this class) and found std::sort.

Reply:
I looked it up and std::sort is the better thing to use normally.